src/components/EntityInfo.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     id="entity-info"
-    :class=" { 'staff': isRoleStaff, 'hover': showHoverStyle }"
+    :class=" { 'staff': accessEntityMenu, 'hover': showHoverStyle }"
```

Comment:
Class "staff" was a neat feature (well I thought so) that I implemented originally but which was never used. I think you can delete it, including the getter and styles on lines 134-138.